public/style.css

```diff
@@ -0,0 +1,48 @@
+#navbar {
+  width: 100%;
+  font-size: 5vh;
+  text-align: center;
+}
+
+.vertical-flex-box {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.horizontal-flex-box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.nav-button {
+  background: none;
+  border: none;
+}
+
+#button-wrapper{
+  display: grid;
+  grid-template-areas: 
+            ". north ."
+            "west . east"
+             ". south .";
+}
+
+#north-button{
+  grid-area: north;
+  width: 50%;
+}
+#south-button{
+  grid-area: south;
+  width: 50%;
+}
+#east-button{
+  grid-area: east;
+  width: 50%;
+}
+#west-button{
```

Comment:
You could condense this a bit by giving the movement buttons a class as well as an id and then just targeting that class to style them all at once.